.github/workflows/regression_test_summariseEpoch.yml

```diff
@@ -0,0 +1,43 @@
+name: Python - summariseEpoch.py
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+   # - name: Lint with flake8
+   #   run: |
```

Comment:
This block can be deleted if we don't need them.

Reply:
Think it would be a good idea to add the linting at some point, once a basic version is working (it's just a bit of work to get the right settings and to run it the first time, so wanted to sort everything else first).